modules/lightmapper_rd/lightmapper_rd.cpp

```diff
@@ -1857,6 +1868,10 @@ LightmapperRD::BakeError LightmapperRD::bake(BakeQuality p_quality, bool p_use_d
 				SWAP(light_accum_tex, light_accum_tex2);
 				error = _denoise(rd, compute_shader, compute_base_uniform_set, push_constant, light_accum_tex2, normal_tex, light_accum_tex, p_denoiser_strength, p_denoiser_range, atlas_size, atlas_slices, p_bake_sh, p_step_function, p_bake_userdata);
 			}
+#else
+			SWAP(light_accum_tex, light_accum_tex2);
```

Comment:
Should put a warning for if denoiser somehow equals 1 that the oidn denoiser can only be used in tool builds.